src/components/Logo.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import { Link } from 'components';
-import { RoadieRIcon } from 'components/icons';
+import { RoadieWordIcon, RoadieRIcon } from 'components/icons';
 
 const Logo = () => (
   <Link to="/">
     <span className="sr-only">Roadie</span>
-    <RoadieRIcon />
+    <span className="md:block hidden">
+      <RoadieWordIcon />
```

Comment:
The full word is what Stripe use by default so I'm just copying that.

I also made everything in the header a little less tall so it doesn't eat so much vertical screen space.